components/Slider/src/MDCSlider.m

```diff
@@ -229,6 +252,14 @@ - (void)updateColorsForState {
   _thumbTrack.trackOffTickColor = [self backgroundTrackTickColorForState:self.state];
 }
 
+- (void)updateThumbForState {
+  if (!self.isStatefulAPIEnabled) {
+    _thumbTrack.thumbRadius = kSliderDefaultThumbRadius;
+  }
```

Comment:
Won't this be overridden by the stateful API value below?

Reply:
Forgot to add the return statement.